R/Reactant/build_tarballs.jl

```diff
@@ -364,7 +364,7 @@ augment_platform_block="""
     """
 
 # for gpu in ("none", "cuda", "rocm"), mode in ("opt", "dbg"), platform in platforms
-for gpu in ("none", "cuda"), mode in ("opt", "dbg"), cuda_version in ("none", "12.1", "12.4", "12.6"), platform in platforms
+for gpu in ("none", "cuda"), mode in ("opt", "dbg"), cuda_version in ("none", "12.1", "12.4", "12.6", "12.8"), platform in platforms
```

Comment:
You'll need to update the platform augmentation file as well.

Reply:
Also https://github.com/JuliaPackaging/Yggdrasil/blob/9828f5f9a130b081a0934751b820cb1b711eb1a8/R/Reactant/platform_augmentation.jl#L95-L101.  And the sources at https://github.com/JuliaPackaging/Yggdrasil/blob/9828f5f9a130b081a0934751b820cb1b711eb1a8/R/Reactant/build_tarballs.jl#L425-L430